### PR TITLE
Update database-engine-service-startup-options.md

### PR DESCRIPTION
**Pull request by pmasl**
Clarifies how to set startup trace flags (no space between -T and number)

### CODE REVIEW

File: docs/database-engine/configure-windows/database-engine-service-startup-options.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Startup options designate certain file locations needed during startup, and spec
...
+|**-T**  *trace#*|Indicates that an instance of [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] should be started with a specified trace flag (*trace#*) in effect. Trace flags are used to start the server with nonstandard behavior. For more information, see [Trace Flags &#40;Transact-SQL&#41;](../../t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md).<br /><br /> **Important:** When specifying a trace flag with the **-T** option, use an uppercase "T" to pass the trace flag number and no space between the -T option and the number of the trace flag. A lowercase "t" is accepted by [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)], but this sets other internal trace flags that are required only by [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] support engineers. (Parameters specified in the Control Panel startup window are not read.)|
```
Comment (rwestMSFT) [inline docs/database-engine/configure-windows/database-engine-service-startup-options.md:63]:
```suggestion
|**-T**  *trace#*|Indicates that an instance of [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] should be started with a specified trace flag (*trace#*) in effect. Trace flags are used to start the server with nonstandard behavior. For more information, see [Trace Flags &#40;Transact-SQL&#41;](../../t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md).<br /><br /> **Important:** When specifying a trace flag with the **-T** option, use an uppercase `T` to pass the trace flag number and no space between the `-T` option and the number of the trace flag. A lowercase `t` is accepted by [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)], but this sets other internal trace flags that are required only by [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] support engineers. (Parameters specified in the Control Panel startup window are not read.)|  
```

**Comment by jborsecnik** [comment]
@ rwestMSFT, 


IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

If you want us to make the changes you requested, please add that in a comment.

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by rwestMSFT** [comment]
@jborsecnik If you can pencil edit my suggested changes, we can get this merged.

**Comment by pmasl** [comment]
@rwestMSFT can this please be merged?

**Comment by rwestMSFT** [comment]
#sign-off

**Pull request merged**